content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+
+Now suppose you also wanted to tag your temperature metric by `state` (which has 2 values: `NY` and `Florida`). You’re now tagging temperature by the set of tags: `country`, `region`, `state`, and `city`. Adding the state tag doesn’t increase the level of granularity already present in your dataset provided by the city tag.
+
+To obtain the temperature in Florida, you can simply recombine the custom metrics of:
+
+* `temperature:{country:USA, state:Florida, city:Orlando}`
+* `temperature{country:USA, state:Florida, city:Miami}`
+
+[1]: /developers/metrics/types/?tab=count#metric-submission-types
+[2]: /developers/metrics/types/?tab=rate#metric-submission-types
+[3]: /developers/metrics/types/?tab=gauge#metric-submission-types
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. [Learn more about HISTOGRAM metric type][1].
+
+Suppose you have a histogram metric measuring `request.Latency` for two endpoints `endpoint:X` and `endpoint:Y`  on a single host:
+
+{{< img src="account_management/billing/custom_metrics/request_latency_single_host.png" alt="request_latency_single_host" responsive="true" style="width:80%;">}}
+
+The number of unique tag value combinations submitted for a `HISTOGRAM` metric with this tagging scheme is 2, but the Agent generates [up to 5 custom metrics][1] for each of the original 2 unique tag value combinations to account [for each Agent-side aggregations enabled][2]: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 10 custom metrics for this `HISTOGRAM` metric: :
+
+| Metric Name       | Tag Values             | Aggregations                            |
+|-------------------|------------------------|-----------------------------------------|
+| `request.Latency` | `host:A`, `endpoint:X` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:A`, `endpoint:Y` | `count`, `sum`, `min`, `max`, and `avg` |
+
+[1]: /developers/metrics/types/?tab=histogram
+[2]: /developers/metrics/types/?tab=histogram#metric-submission-types
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the server-side aggregations  `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
+
+Suppose you have a distribution metric measuring `request.Latency` in which the following tag value combinations are present in your data as shown below:
+
+{{< img src="account_management/billing/custom_metrics/request_latency.png" alt="Request latency" responsive="true" style="width:80%;">}}
+
+The number of unique combinations of metric name, host, and tag values submitted for a `GAUGE` metric with [this tagging scheme is 4](#counting-custom-metrics?tab=countrategauge):
```

Comment:
Given [this same tagging scheme], the number of unique combinations of metric and tag values submitted for a `GAUGE` metric is 4(#counting-custom-metrics?tab=countrategauge):